java/arcs/core/storage/Store.kt

```diff
@@ -81,6 +81,8 @@ class Store<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
         return activeStore
     }
 
+    suspend fun waitForActiveIdle() = activeStore?.idle()
```

Comment:
The return type of this is currently `Unit?`. Might be a little cleaner with the `activeStore?.idle()` within normal braces here, so the return type is `Unit`.

Reply:
Good catch; done.